ecosystem/sep-0024.md

```diff
@@ -583,6 +584,7 @@ Name | Type | Description
 `status` should be one of:
 
 * `incomplete` -- there is not yet enough information for this transaction to be initiated. Perhaps the user has not yet entered necessary info in an interactive flow.
+* `pending_kyc` -- the user has submitted their KYC information and the anchor is verifying it.
 * `pending_user_transfer_start` -- the user has not yet initiated their transfer to the anchor. This is the necessary first step in any deposit or withdrawal flow.
 * `pending_external` -- deposit/withdrawal has been submitted to external network, but is not yet confirmed. This is the status when waiting on Bitcoin or other external crypto network to complete a transaction, or when waiting on a bank transfer.
 * `pending_anchor` -- deposit/withdrawal is being processed internally by anchor. This can also be used when the anchor must verify KYC information prior to deposit/withdrawal.
```

Comment:
Are we keeping the KYC language on `pending_anhcor`?

Reply:
It does sound like pending_anchor already does the thing we want.

Reply:
Good catch Tomer. This second sentence should be removed, however existing anchors will likely continue to use `pending_anchor` while verifying KYC. No change for them is needed unless they work with a client that wants to see the `pending_kyc` status.

Reply:
@leighmcculloch it doesn't because its a wider umbrella then just KYC. SEP-24 needs have a way to communicate that the anchor is specifically working on KYC validation

Reply:
@JakeUrban, what other states are under that wider umbrela of `pending_anchor`?

Reply:
`pending_anchor` is not completely covered by sub-statuses, really there is no such thing other than this one case.

Meaning, `pending_anchor` historically has been a catch-all for "things the anchor needs to do that it doesn't have a better status for". Now that there's `pending_kyc`, the anchor should use that whenever its relevant in place of `pending_anchor`, however `pending_anchor` will still be necessary for other situations.

Reply:
There are no other statuses that could be seen as substitutes for. `pending_anchor`. Its possible some anchors use `pending_anchor` when really its `pending_external`, but thats a minor thing and has no affect on functionality.

Reply:
@JakeUrban can you give some examples of these other situations that `pending_anchor` will be necessary for?

It is my understanding that this PR attempts to de-complect kyc verification from other potential scenarios in which the wallet is waiting on the anchor. I just don't have a full understanding of what those scenarios are. 

My mental model of `pending_anchor` has been that someone needs to take a look at the KYC information, transaction details and make a yes/no decision. If you take that way from `pending_anchor` - what is left? 

This is a breaking change. We need to have strong conviction here.

Reply:
We'll use `pending_anchor` for now.

Reply:
Other than KYC, `pending_anchor` is used when 

- Polaris is constructing the Stellar transaction envelope to be submitted to the network
- Polaris has detected a withdrawal payment to its distribution account and is sending the funds off chain

Directly after the first case listed, the status will be `pending_stellar`. Directly after the second case listed, the status will be `pending_external` if still pending or `completed` if the user received the funds off-chain.

Reply:
> Polaris has detected a deposit payment to the anchor's bank account and is constructing the Stellar transaction envelope to be submitted to the network

Reply:
Jake, this is super helpful. In both these specific cases it sounds like `pending_anchor` is a very short ephemeral status - can we just skip it?

Though I imagine that with multi-sig distribution transactions your first example can take a longer...

Reply:
> can we just skip it?

From a UX perspective, the same rationale could be used for `pending_stellar`, since it's also very temporary. The answer depends on if you want clients or users to know either of those two things happen. If not, you can just notify of success or failure when it happens. Until then, it would be in `pending_user_transfer_start`.

If the transaction requires signatures from distributed systems, then being `pending_anchor` is much more significant since it could take a long time.

Reply:
To get this moving, I removed pending_kyc. Anchors that don't support any change can use `pending_anchor` for now.